app/src/app/[lng]/onboarding/setup/page.tsx

```diff
@@ -819,22 +204,13 @@ export default function OnboardingSetup({
     }
   };
 
-  const onSubmit: SubmitHandler<Inputs> = async (newData) => {
-    const year = Number(newData.year);
-
+  const onSubmit: SubmitHandler<Inputs> = async (formData) => {
     setData({
-      name: newData.city,
-      locode: data.locode!,
-      year,
+      ...data,
+      ...formData,
+      locode: ocCityData?.actor_id!,
+      name: ocCityData?.name!,
     });
-
-    if (!newData.city || !ocCityData?.actor_id || year < 0 || !data.locode) {
-      // TODO show user toast? These should normally be caught by validation logic
-      makeErrorToast("Missing data, can't go to next step!");
-      console.error("Missing data, can't go to next step!");
-      return;
-    }
```

Comment:
Why did you remove the validation here? Is it handled later?

Reply:
Yes its being handled during the process.